config/webpack/plugin/SetupPlugin.js

```diff
@@ -64,26 +64,11 @@ class SetupPlugin {
             // Use MDX to import any md files imported from an mdx file.
             issuer: [
               /\.mdx?$/,
-            ],
-            use: [
-              babelLoader,
-              mdxLoader,
-            ],
-          },
-          {
-            // Use the marked loader to load any md files included by the content config file.
-            // LOAD USING MDX ON NEXT MAJOR VERSION.
-            issuer: [
               /dev-site-config.*[/\\]contentConfig\.js$/,
             ],
             use: [
               babelLoader,
-              {
-                loader: 'devSiteMarked',
-                options: {
-                  baseUrl: this.publicPath,
-                },
-              },
+              mdxLoader,
```

Comment:
does this improve compile time?

Reply:
I wouldn't expect a significant increase, we're trading one loader for another.

Reply:
I also didn't collect the before and after.

Reply:
Very Roughly the terra-dev-site `lint jest and compile` stage went from 6 minutes 28 seconds in main to 6 minutes and 21 seconds in the PR branch. So I think thats within tolerances.